torch/csrc/jit/operator.cpp

```diff
@@ -160,11 +163,8 @@ struct SchemaParser {
   }
 
   IValue parseTensorDefault(const SourceRange& range) {
-    if("None" == L.expect(TK_IDENT).text()) {
-      return at::Tensor();
-    } else {
-      throw ErrorReport(range) << "invalid tensor default value";
-    }
+    L.expect(TK_NONE);
+    return at::Tensor();
```

Comment:
This can `return IValue()`. We always change it to the right type in operator dispatch.